### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'write'.

### ISSUE THREAD

**Issue by JoseHenriqueSiqueira** [marcelotduarte/cx_Freeze#1696]
When I create an executable that contains the module 'webdriver_manager' it ends up returning this error "'NoneType' object has no attribute 'write'" when I open the executable. Apparently this happens when the driver is going to be downloaded. If the driver is already downloaded, it works normally. I tested it in other scripts and it always returns the same error.
NOTE: I used the webdriver for CHROME.

![printt](https://user-images.githubusercontent.com/116378234/204930812-fbceaeaa-5661-4a73-a254-1ac8bef07be3.png)

Teste.py
```python
from webdriver_manager import chrome
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from time import *
import os

def checkdriver():
    wddir = os.environ["USERPROFILE"] + "\.wdm\drivers\chromedriver\win32"
    if os.path.exists(wddir):
            wdname = "chromedriver.exe"
            for root, dir, files in os.walk(wddir):
                if wdname in files:
                    SERVICE=Service(os.path.join(root, wdname))
    else:
            SERVICE=Service(chrome.ChromeDriverManager().install())
    driver=webdriver.Chrome(service=SERVICE)
    driver.get("https://www.google.com/")
    driver.execute_script("alert('OK')")
    sleep(4)


if __name__ == "__main__":
    checkdriver()
    input("\n------FIM------\nPRESSIONE ALGUMA TECLA PARA FINALIZAR")
```

setup.py

```python
import sys
from cx_Freeze import setup, Executable

base = None
if sys.platform == "win32":
    base = "Win32GUI"

setup(
    name="Teste",
    version="1.0",
    description="Teste",
    options = {
    'build_exe': {
        'zip_include_packages':['selenium']
    }},
    executables=[Executable("Teste.py", base=base)]
)

**Comment by WillyJL** [comment]
Using the win32gui base removes the console input and output from you program, and some of the code you're using is trying to write stuff out to the console so it fails. Try leaving the base as None

**Comment by JoseHenriqueSiqueira** [comment]
I tried your suggestion, but it didn't work. I decided to take another look at the Python files and noticed that CX_Freeze was getting the 'webdriver_manager' in the Python 3.1x directory. However, I currently use Python 3.9. I removed that 3.1x version, but for some reason Cx_Freeze was trying to get the module from that version. So I deleted that folder (folder with remnants of python 3.1x), compiled again and it worked.
